ppr-ui/src/components/common/SimpleHelpToggle.vue

```diff
@@ -34,19 +35,22 @@
 </template>
 
 <script lang="ts">
+import { computed } from 'vue'
 import { defineComponent, reactive, toRefs } from 'vue-demi'
 
 export default defineComponent({
-  name: 'SimleHelpToggle',
+  name: 'SimpleHelpToggle',
   props: {
     toggleButtonTitle: { default: '' },
     /* show or hide secondary toggle within content */
-    hasBottomHideToggle: { default: true }
+    hasBottomHideToggle: { default: true },
+    defaultHideText: { default: true }
   },
   setup (props) {
     const localState = reactive({
       isHelpContentOpen: false,
-      title: props.toggleButtonTitle,
+      hideText: props.defaultHideText ? 'Hide Help' : 'Hide ' + props.toggleButtonTitle,
```

Comment:
So now, as the `defaultHideText` is true by default, it will always have 'Hide Help'?

Reply:
Yes, if not specified `defaultHideText` false.

Reply:
In the current design for this flow the
`toggleButtonTitle` is: Need Help? Contact Us

![image](https://github.com/bcgov/ppr/assets/77707952/cd94b52c-3b97-4e32-b1f0-2a1ac5f3ccfa)

The Hide text is:

![image](https://github.com/bcgov/ppr/assets/77707952/2a0a71ec-9988-477c-a015-b4e5c9fe18bf)

So I don't think it make sense to check if toggleButtonTitle.

I can check the prop name to be something other than default. 

Maybe `genericHideText` and then set it to false by default.